SETUP.md

```diff
@@ -138,8 +138,8 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 
     1. Pick up those changes: `source ~/.bash_profile`
 1. Install Node and yarn
-    1. `nvm install 8.15.0 && nvm alias default 8.15.0` this command should make this version the default version and print something like: `Creating default alias: default -> 8.15.0 (-> v8.15.0)`
-    1. `npm install -g yarn@1.16.0`.
+    1. `nvm install 14.17.1 && nvm alias default 14.17.1` this command should make this version the default version and print something like: `Creating default alias: default -> 14.17.1 (-> v14.17.1)`
+    1. `npm install -g yarn@1.22.10`.
```

Comment:
non-blocking comment: Just a note that yarn v1.22.11 dropped yesterday. Perhaps you can include bumping this in the documentation you are writing and then catch it on the node 16 upgrade?